.openshift/cron/hourly/git_repos

```diff
@@ -29,7 +29,6 @@ curl http://prprocessor-theforeman.rhcloud.com/redmine_repos | ruby -rjson -e '
 JSON.load(STDIN).each do |project_name,repos|
   repos.each do |repo,branches|
     org_name, repo_name = repo.split("/", 2)
-    repo_path = File.join(ENV["OPENSHIFT_DATA_DIR"], "git", repo_name, ".git")
```

Comment:
I'm pretty sure this line is unused but please verify.

Reply:
Yeah I don't think so either